Cartfile.resolved

```diff
@@ -9,7 +9,7 @@ github "Udumft/SnappyShrimp" "66f3e3ba70370ad5e889ac8ed72a8730ca79958e"
 github "ceeK/Solar" "2.1.0"
 github "linksmt/OHHTTPStubs" "563f48d3fab84ef04639649c770b00f4fa502cca"
 github "mapbox/MapboxGeocoder.swift" "v0.10.2"
-github "mapbox/mapbox-directions-swift" "v0.33.1"
+github "mapbox/mapbox-directions-swift" "v0.33.2"
```

Comment:
This is fine, but FYI it isn’t strictly necessary to require v0.33.2 as opposed to v0.33.1, since v0.33.2 is just a bug fix release without any API changes.